LICENSES/README

```diff
@@ -0,0 +1,60 @@
+The systemd project uses a single line reference to Unique License Identifiers
+in source files as defined by the Linux Foundation's SPDX project
+(https://spdx.org/).
+
+The current set of valid, predefined SPDX identifiers can be found on the SPDX
+License List at https://spdx.org/licenses/.
+
+The 'license/' directory contains all the licenses used by the sources included
```

Comment:
Did you mean `LICENSES/` directory here?

Is it deliberate to have a `license` and `LICENSES` directory now?

Reply:
nope, just a typo

Reply:
Ah, ok. I was also confused a bit by the commit message which says "add license/README explaining the license situation"

Reply:
yeah I renamed it half way through and forgot a couple of things